RecoLocalMuon/RPCRecHit/src/CSCSegtoRPC.cc

```diff
@@ -128,12 +158,12 @@ CSCSegtoRPC::CSCSegtoRPC(edm::Handle<CSCSegmentCollection> allCSCSegments, const
 	  float dz=segmentDirection.z();
 
 	  if(debug)  std::cout<<"Calling to Object Map class"<<std::endl;
-	  ObjectMapCSC* TheObjectCSC = ObjectMapCSC::GetInstance(iSetup);
+	  //ObjectMapCSC* TheObjectCSC = new ObjectMapCSC(iSetup);
```

Comment:
Why is this comment still here? Uncommenting the line would be a major change to  the interface of this constructor.